bin/runtime/src/lib.rs

```diff
@@ -1478,6 +1491,7 @@ mod tests {
                 change_rate: _,
             } => {}
             pallet_nomination_pools::Call::claim_commission { pool_id: _ } => {}
+            pallet_nomination_pools::Call::adjust_pool_deposit { .. } => {}
```

Comment:
This is probably easy to miss, so in case you missed it -- see the comment in `test_call_runtime_api_stability`. 
Also I will accept any suggestions to make this hard to miss, or make this test better :P

Reply:
Tbh I think this test is not that needed. Any breaking change should be catched up by one of the 2 tools we have to inspect runtime changes and/or should also be described in polkadot-sdk releases, since well they are breaking. We could add checks to the ci if the given commit introduces such changes using one of the tools we have but I think there would be a lot of false positives

Reply:
If we are worried about, allowing more calls in the proxy accounts that intended we should add action point to the releases to check it before relaesing

Reply:
The point of this test is to catch such breaking changes very early, and not as the last check before the release, because then it's kind of too late. The reason why we want to catch it early is because `call_runtime` on contracts makes this extra sensitive, to the extent that some such changes we will not be able to merge at all. 

I'm not sure what the best approach here should be TBH, but my concern is that if we wait till release checks, then such changes could be already buried under dozens of other commits and it will be harder to react.